packages/page-accounts/src/modals/Create.spec.tsx

```diff
@@ -0,0 +1,128 @@
+// Copyright 2017-2021 @polkadot/page-accounts authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { fireEvent, screen, waitForElementToBeRemoved } from '@testing-library/react';
+
+import i18next from '@polkadot/react-components/i18n';
+import { MemoryStore } from '@polkadot/test-support/keyring';
+import { keyring } from '@polkadot/ui-keyring';
+
+import { AccountsPage } from '../../test/pages/accountsPage';
+
+const spy = jest.spyOn(keyring, 'addUri');
+
+const newAccountName = 'NEW ACCOUNT NAME';
+const newAccountPassword = 'mySecretPassword';
+
+describe('Create an account modal', () => {
+  let accountsPage: AccountsPage;
+
+  beforeAll(async () => {
+    await i18next.changeLanguage('en');
+    keyring.loadAll({ isDevelopment: true, store: new MemoryStore() });
+  });
+  beforeEach(() => {
+    accountsPage = new AccountsPage();
+  });
+
+  it('creates an account', async () => {
+    await accountsPage.enterCreateAccountModal();
+
+    await screen.findByText('Add an account via seed 1/3');
```

Comment:
maybe `await screen.findByText('Add an account via seed 1/3');` could be included in enterCreateAccountModal()? It would reduce the test code lines and we can assume that the modal is entered if that text is rendered

Reply:
✅